RFCs/0002-add-pdk-config.md

```diff
@@ -109,9 +111,35 @@ Ref: (Verb-Noun Reference)[https://docs.google.com/document/d/1zX0FJBAvAIK3d3L3Q
       - If `<value>` is already present in the value list, issues a notice that value was already on list, outputs the
         current values of `<key>`, and exits with a success status.
 
-    If `<key>` is a user-defined configuration key where we cannot infer whether or not it can have multiple values,
-    assume it is a single-value key unless the user includes the `--add` flag. If the user includes the `--add` flag,
-    treat it as a multi-value key as described above.
+    If the `--force` flag is set then the `<key>` will be literally set to `<value>`.  That is, list modifications like above will not be used.
```

Comment:
If I say `--force` on a setting expected to be (or currently) an array, but pass in an untyped value (or a typed but non-array value) what is the behavior?

Reply:
It will attempt to overwrite it with the literal value.  So if you force a boolean to something that is an array, it will become a boolean.  It's a way to short-circuit any "magic" munging the PDK does and "just do it"

This will normally cause an error if the setting has validation and a schema.  However for settings files with no schema, it's the Wild West when it comes to setting types.

Reply:
Gotcha, so `--force` will bypass any automatic "string to single-item array" conversion or anything like that, and in the case where we have a schema for the setting, it will error out if the types don't match. That sounds good.

Reply:
I'm open to different name for it.  e.g. `--overwrite`, `--literal` etc.  Force seemed the most appropriate and is already used in the PDK.

Reply:
I think `--force` is fine.